### PR TITLE
Fix for panic when user/settings watch connection closes

### PR DESCRIPTION
**Pull request by lsviben**
**What this PR does / why we need it**:
fix for the panic that happens on user/settings watch connection close

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/handler/websocket/common.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import (
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// writes close control message to the websocket connection. Same as the default websocket close handler.
+func writeCloseMessage(ws *websocket.Conn, code int) {
+	message := websocket.FormatCloseMessage(code, "")
+	_ = ws.WriteControl(websocket.CloseMessage, message, time.Now().Add(time.Second))
```
Comment (floreks) [inline pkg/handler/websocket/common.go]: I guess this returns potential err, wouldn't it be better to return it instead of returning `nil`?

Comment (lsviben) [reply pkg/handler/websocket/common.go]: you are right, i did it automatically like websocket does it, returning err is better

**Pull request merged**